garrysmod/lua/autorun/properties/editentity.lua

```diff
@@ -19,9 +19,18 @@ properties.Add( "editentity", {
 
 	Action = function( self, ent )
 
+		local printName = ent.PrintName
+
+		if printName then
+			printName = language.GetPhrase( printName )
+			printName = string.format( "%s [%d]", printName, ent:EntIndex() )
+		else
+			printName = tostring( ent )

```

Comment:
This could go through `language.GetPhrase` as well, some entities add their class as the localization string.

Reply:
I changed that it falls back to the entity classname now. The both cases are GetPhrase'ed now.

Reply:
I meant use the class instead of print name if there's no print name, sorry.

Reply:
Yeah, someone pointed that out on Discord already. 😅